config/manager/persistent-volume-claim.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: repo-pvc
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteOnce
```

Comment:
I wonder if we need `ReadWriteMany`? I see "ReadWriteOnce" means "the volume can be mounted as read-write by a single node. ReadWriteOnce access mode still can allow multiple pods to access the volume when the pods are running on the same node." We will have one Leader Controller Pod, and one Follower Controller Pod, and they can be on different nodes.

Reply:
It seems fine as we should only read/write from the leader at a time?

Reply:
I'm not sure? Is it?